prac_05/hex_colours.py

```diff
@@ -0,0 +1,15 @@
+COLOUR_LIST = {'aliceblue': '#f0f8ff', 'antiquewhite': '#faebd7', 'antiquewhite1': '#ffefdb',
+               'antiquewhite2': '#ffefdb', 'antiquewhite3': '#cdc0b0', 'antiquewhite4': '#8b8378',
+               'aquamarine1': '#7fffd4', 'aquamarine2': '#76eec6', 'aquamarine4': '#458b74', 'azure1': '#f0ffff'}
+
+
+def main():
```

Comment:
The code looks great but once again I think comments could be added including a doc string at the start